test/disseminator_tests.ml

```diff
@@ -13,23 +13,23 @@ module Disseminator_tests = struct
 
   let queue_insertion_test () =
     let _server = Node.run_server ~msg_handler:Commons.msg_handler node in
-    Client.address_of !node
-    |> (fun Address.{ port; _ } -> port)
-    |> string_of_int
-    |> String.to_bytes
-    |> Client.create_post node
-    |> Client.post node;
+    let payload =
+      Client.address_of !node
+      |> (fun Address.{ port; _ } -> port)
+      |> string_of_int
+      |> String.to_bytes in
+    Client.create_post node payload ?payload_signature:None |> Client.post node;
 
     Lwt.return (List.length (Node.Testing.broadcast_queue node))
 
   let queue_removal_test () =
     let _server = Node.run_server ~msg_handler:Commons.msg_handler node in
-    Client.address_of !node
-    |> (fun Address.{ port; _ } -> port)
-    |> string_of_int
-    |> String.to_bytes
-    |> Client.create_post node
-    |> Client.post node;
+    let payload =
+      Client.address_of !node
+      |> (fun Address.{ port; _ } -> port)
+      |> string_of_int
+      |> String.to_bytes in
+    Client.create_post node payload ?payload_signature:None |> Client.post node;
```

Comment:
No need for specifying None here, it's an optional argument so not specifying it at all will default to None